### DISCUSSION TITLE
Updating links on grid shift error page

### DISCUSSION THREAD

**Discussion by alexruudrespec** [DotSpatial/DotSpatial#1492] [General]
Hello there!

No idea if this is the correct place to post this, so hopefully someone can notify the proper authorities if not!

I come to you after receiving the grid shift file exception as I was testing projections. This is not a complaint, however. On the contrary, it was extremely helpful to be immediately notified of the problem _and_ a way to address it! I spent 4 days debugging ProjNET before figuring out that it doesn't support datum transformations because it doesn't give any kind of error when doing so.

It seems the page that the grid shift exception points to ( https://github.com/DotSpatial/DotSpatial/wiki/Grid-Shifts ) could use a touch of updating, however. The first link ( https://proj.org/wiki/GenParms#nadgrids-GridBasedDatumAdjustments ) now 404s, which I assume was a primer on grid shift files. The third link ( https://github.com/DotSpatial/DotSpatial/tree/master/Source/DotSpatial.Projections/Nad/GridShifts ) also 404s, but it simply needs updating to the new location (https://github.com/DotSpatial/DotSpatial/tree/master/Source/Core/DotSpatial.Projections/Nad/GridShifts or perhaps https://github.com/DotSpatial/DotSpatial/tree/master/Source/Core/DotSpatial.Projections/Nad/ to let users know there is a bundled option?).

The second link is still good and a refreshingly frank discussion. Thanks for creating such an easy library to use! I'm off to figure out which grid shift files I need and how to get them working!

Cheers,
-Alex

**Comment by jany-tenaj** [comment]
urls should be working now
